### PR TITLE
Add wagtail model admin

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

**Comment by saeedmb8** [comment]
@miltontony tiny review :)

File: molo/core/models.py [hunk 1]
```diff
@@ -114,8 +114,16 @@ class LanguagePage(CommentedPageMixin, Page):
         max_length=255,
         help_text=_('The language code as specified in iso639-2'))
 
+    TYPE_CHOICES = (
+        ('P', 'Parent'),
+        ('c', 'Child'),
+    )
+    type = models.CharField(
```
Comment (miltontony) [inline molo/core/models.py]: Can we make this a BooleanField `is_main_language`

**Comment by saeedmb8** [comment]
@miltontony ready for review :)

**Comment by miltontony** [comment]
:+1:

**Pull request merged**